test/verify/check-menu

```diff
@@ -44,6 +45,32 @@ class TestMenu(MachineCase):
         b.click('#about button[data-dismiss="modal"]')
         b.wait_popdown("about")
 
+        # Test session timeout
+        time.sleep(10)
+        with b.wait_timeout(1):
+            b.wait_not_visible("#session-timeout-dialog")
+            b.wait_visible("#filter-menus")
+            b.enter_page("/system")
+            b.mouse("#system_information_hardware_text", "mousemove", 24, 24)
+            b.switch_to_top()
```

Comment:
Why does this need to run with such a short timeout? This sounds like a recipe for flaky tests.

Reply:
I wanted to have more control over the test since this is timing specific. So I want to make sure that moving mouse actually resets the timer - with default  timeout it is hard to check if the timer was actually reset by mouse moving, or just it took too long to check for presence of some element.